### DISCUSSION TITLE
How can I allow sandbox to modify host files?

### DISCUSSION THREAD

**Discussion by zeymx** [sandboxie-plus/Sandboxie#2870] [Documents and notes]
I'm a little bit confused with UI and I find only the setting to allow reading from drive, not writing. Asking for help

**Comment by offhub** [comment]
**Open**: This allows write-access outside the sandbox.
**Open for All**: This allows write-access outside the sandbox, also for applications installed inside the sandbox.

https://sandboxie-plus.github.io/sandboxie-docs/Content/OpenFilePath.html

**Reply by zeymx** [reply]
I can't find anything related to "open" and "open for all" in this article...

**Reply by offhub** [reply]
**Normal** is `NormalFilePath` (Regular Sandboxie behavior - allow read and also copy on write).
**Open** is `OpenFilePath` (Allow write-access outside the sandbox).
**Open for All** is `OpenPipePath` (Allow write-access outside the sandbox, also for applications installed inside the sandbox).
**Read Only** is `ReadFilePath` (Allow read-only access only).
**Closed** is `ClosedFilePath` (Deny access to host location and prevent creation of sandboxed copies).
**Box Only** is `WriteFilePath` (Hide host files, folders from sandboxed processes).

**Reply by bastik-1001** [reply]
Here is a [comparison of resource access rules](https://sandboxie-plus.github.io/sandboxie-docs/Content/ResourceAccess.html), just in case, someone needs references for other resource types.
